pages/resources/safety-hub.mdx

```diff
@@ -0,0 +1,398 @@
+| account_standing      | [account standing](#account-standing-state) object     | Current standing of the user's account                                 |
+| is_dsa_eligible       | boolean                                                | Indicates if the user is eligible for DSA appeal                       |
+| is_appeal_eligible    | boolean                                                | Indicates if the user is eligible to appeal any classifications        |
+| username              | string                                                 | The username of the user                                               |
+| appeal_eligibility    | array[integer]                                         | List of [appeal eligibility](#appeal-eligibility) reasons for the user |
+
+### Classification Object
+
+###### Classification Structure
+
+| Field                 | Type                                                     | Description                                                        |
+| --------------------- | -------------------------------------------------------- | ------------------------------------------------------------------ |
+| id                    | snowflake                                                | The ID of the classification                                       |
+| classification_type   | integer                                                  | The [classification type](#classification-type)                    |
+| description           | string                                                   | A brief description of the classification                          |
+| explainer_link        | string                                                   | URL to a detailed explanation of the classification                |
+| actions               | array[[action](#actions-object) object]                  | List of actions taken as part of the classification                |
+| max_expiration_time   | ?ISO8601 timestamp                                       | The time at which the classification will expire                   |
+| flagged_content       | array[[flagged content](#flagged-content-object) object] | List of messages that was flagged as part of the classification    |
+| appeal_status?        | [appeal status](#appeal-status) object                   | Status of an appeal related to the classification                  |
+| is_coppa              | boolean                                                  | Whether the classification is related to COPPA                     |
+| is_spam               | boolean                                                  | Whether the classification is related to spam                      |
+| appeal_ingestion_type | integer                                                  | How a nominal appeal should be submitted                           |
```

Comment:
add a footnote to this and the above 2 fields outining the client behavior around appeals that we discussed